Source/WebCore/loader/cache/CachedResourceLoader.cpp

```diff
@@ -1494,6 +1494,11 @@ CachedResourceLoader::RevalidationPolicy CachedResourceLoader::determineRevalida
         return Reload;
     }
 
+    // FIXME: We should be able to USE the data here, but cannot currently due to a bug
+    // concerning redirect and URL fragments.
```

Comment:
Can we file a bugzilla and add the bug URL to this FIXME.

Reply:
Done. Will add here https://bugs.webkit.org/show_bug.cgi?id=258934